src/index.js

```diff
@@ -0,0 +1,22 @@
+import './styles.css';
+import changeValue from './modules/store.js';
+import deleteTask from './modules/delete.js';
+import editTask from './modules/editTask.js';
+import { checkUpdate } from './modules/check.js';
+import clear from './modules/removeAll.js';
+
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+if (todos.length !== 0) {
+  editTask();
+  deleteTask();
+}
+
+window.addEventListener('load', () => {
+  todos.forEach((e) => {
+    checkUpdate(e);
+  });
+});
+
+clear();
+changeValue();
```

Comment:
- [ ] Good job so far :+1: But unfortunately your todo list is not working as expected as you can see on the video I'm sharing with you below :arrow_down:  Please make sure that Implement a function for adding a new task, a function for deleting a task and a function for editing task descriptions that are working as expected :+1: 

You can check the video below :arrow_down: 
https://www.loom.com/share/0a42f64ea6e848ddaf367c86d46bc4eb

- [ ] According to the project [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md), **By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).** Please fix it :+1: 

- [ ] One more this mentioned in the project requirement is: **Deleting a task should update all remaining items' indexes, so they represent the current list order and are unique(i.e. if you're deleting the first task index 1 from the list, the index of the next task(2) should set to 1).** with your project which is not working as expected we can not check, please fix it :+1: 
- [ ] As you can see in the video I'm sharing above arrow_up:  saving in local storage is not working please make sure that all changes to the To Do List are saved in local storage as expected , please fix it :+1:

Reply:
Thank you very much for your valuable opinion. Ultimately today my previous project has been approved and the To Do List is now working. Now I will make the necessary changes in add and remove part (this one) and will make the request review as early as possible. I was waiting for this project's approval. I am attaching the To-Do List here for your reference.

https://user-images.githubusercontent.com/106825373/194594954-09a662cf-3c94-4a96-a19c-df0fb722f758.png